### ISSUE TITLE
schema-reader-db Error: Failed opening required '/Users/patrickbergman/Herd/project/bootstrap/../routes/app.php' (include_path='.:')

### ISSUE THREAD

**Issue by patrickbergman** [VemtoOrg/vemto2-issues#200]
**Error Message:**
schema-reader-db Error: Failed opening required '/Users/patrickbergman/Herd/project/bootstrap/../routes/app.php' (include_path='.:')

**Stack:**
```
schema-reader-db Error: #0 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Routing/Router.php(513): Illuminate\Routing\RouteFileRegistrar->register('/Users/patrickb...')
#1 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Routing/Router.php(467): Illuminate\Routing\Router->loadRoutes('/Users/patrickb...')
#2 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php(194): Illuminate\Routing\Router->group(Array, '/Users/patrickb...')
#3 /Users/patrickbergman/Herd/project/bootstrap/app.php(18): Illuminate\Routing\RouteRegistrar->group('/Users/patrickb...')
#4 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Configuration/ApplicationBuilder.php(232): {closure}(Object(Illuminate\Foundation\Application))
#5 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\Foundation\Configuration\ApplicationBuilder->Illuminate\Foundation\Configuration\{closure}()
#6 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/Util.php(43): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#7 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(83): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#8 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure), Object(Closure))
#9 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/Container.php(690): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Object(Closure), Array, NULL)
#10 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Support/Providers/RouteServiceProvider.php(162): Illuminate\Container\Container->call(Object(Closure))
#11 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Support/Providers/RouteServiceProvider.php(59): Illuminate\Foundation\Support\Providers\RouteServiceProvider->loadRoutes()
#12 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(36): Illuminate\Foundation\Support\Providers\RouteServiceProvider->Illuminate\Foundation\Support\Providers\{closure}()
#13 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/Util.php(43): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#14 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(83): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#15 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php(35): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Object(Closure), Object(Closure))
#16 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Container/Container.php(690): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Object(Closure), Array, NULL)
#17 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Support/ServiceProvider.php(130): Illuminate\Container\Container->call(Object(Closure))
#18 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1122): Illuminate\Support\ServiceProvider->callBootedCallbacks()
#19 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1100): Illuminate\Foundation\Application->bootProvider(Object(Illuminate\Foundation\Support\Providers\RouteServiceProvider))
#20 [internal function]: Illuminate\Foundation\Application->Illuminate\Foundation\{closure}(Object(Illuminate\Foundation\Support\Providers\RouteServiceProvider), 'Illuminate\\Foun...')
#21 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(1099): array_walk(Array, Object(Closure))
#22 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/BootProviders.php(17): Illuminate\Foundation\Application->boot()
#23 /Users/patrickbergman/Herd/project/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(316): Illuminate\Foundation\Bootstrap\BootProviders->bootstrap(Object(Illuminate\Foundation\Application))
#24 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/index.php(39): Illuminate\Foundation\Application->bootstrapWith(Array)
#25 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/common/Vemto.php(111): {closure}()
#26 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL2/index.php(20): Vemto\Vemto::execute('schema-reader-d...', Object(Closure))
#27 /Applications/Vemto.app/Contents/Resources/static/VMTTL2(15): require('phar:///Applica...')
#28 {main}
```

**How to reproduce:**
1. Create a new Laravel app via Vemto
2. Add a new table plus model
3. Click Save

**Expected behavior:**
Save the app and models
**Actual behavior:**
Above error message
**Additional context:**
I run everything with the latest version
---

Vemto 2.1.4
darwin 23.6.0
Locale: en-US
App version: 2.1.4

**Comment by TiagoSilvaPereira** [comment]
Hi @patrickbergman I just fixed this on Vemto 2.1.5.

If you want to reuse the same project, please create two empty PHP files inside the /routes folder:

app.php
app-api.php

Or create a new project.

**Comment by patrickbergman** [comment]
Thank you very much. It seems to work :)
